test/e2e/etcd_multi_node_test.go

```diff
@@ -105,6 +106,19 @@ var _ = Describe("Etcd", func() {
 			// K8s job zeroDownTimeValidator will fail, if there is any downtime in Etcd cluster health.
 			checkEtcdZeroDownTimeValidatorJob(ctx, cl, client.ObjectKeyFromObject(job), objLogger)
 
+			By("Single member restoration")
+			objLogger.Info("Delete one member pod")
+			deletePodAndCheckSts(ctx, cl, logger, etcd, "etcd-aws-2")
```

Comment:
IMO, deletion of a pod won't affect anything ... as pvc/data-dir is intact. 
Is this tests deleting the PVC/data-dir if it is then fine else IMO it is not so useful. WDYT?

Reply:
No hard opinion. Just added this test to cover all cases
Would you rather I remove this test?

Reply:
ok, we can keep this tests but don't include this in `Single member restoration` as single member restoration is not doing anything in this case and don't call it as `Delete one member pod`, you can call it as `restart of member with data-dir/pvc intact` as it is similar to transient down of 1 member of a cluster.

Reply:
Moved this to a new block